docs/book/src/clusterctl/commands/alpha-rollout.md

```diff
@@ -0,0 +1,51 @@
+# clusterctl alpha rollout
+
+The `clusterctl alpha rollout` command manages the rollout of a cluster-api resource. It consists of several sub-commands which are documented below. 
+
+<aside class="note">
+
+<h1> Valid Resource Tpes </h1>
+
+Currently, only the following cluster API resources are supported by the rollout command:
+
+- machinedeployment
+
+</aside>
+
+### Restart 
+
+Use the `restart` sub-command to force an immediate rollout. For example, here the MchineDeployment `my-md-0` will be immediately rolled out:
+
+```
+clusterctl alpha rollout restart machinedeployment/my-md-0
+```
+
+### Undo
+
+Use the `undo` sub-command to rollback to an ealier revision. For example, here the MachineDeployment `my-md-0` will be rolled back to revison number 3. If the `--to-revision` flag is omited, the MachineDeployment will be rolled back to the revision immediately preceding the current one. 
+
+```
+clusterctl alpha rollout undo machinedeployment/my-md-0 --to-revision=3`
+```
+
+### Pause/Resume
+
+Use the `pause` sub-command to pause a Cluster API resource. 
+
+```
+clusterctl alpha rollout pause machinedeployment/my-md-0
+```
+
+Use th `resume` sub-command to resume a currently paused Cluster API resource. 
+
+```
+clusterctl alpha rollout resume machinedeployment/my-md-0
+```
+
+<aside class="note warning">
+
+<h1> Warning </h1>
+
+Paused resources will not be reconciled by a controller. By resuming a resource, we allow it to be reconciled again. 
```

Comment:
Are we using paused at object level or machineDeployments.spec.Paused?
In case we are using the first, should we document that pausing could imply also some undesired side effects because nothing gets updated, including e.g. the ObservedGeneration, replica counters etc.